src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/InnerAssignmentCheckTest.java

```diff
@@ -36,7 +36,7 @@ protected String getPackageLocation() {
     }
 
     @Test
-    public void testIt() throws Exception {
+    public void testInputInnerAssignmentCheckLocalVariables() throws Exception {
```

Comment:
Not done :(
Please check my comment at https://github.com/checkstyle/checkstyle/pull/14543#discussion_r1502054425

test method names should not have "`Input`" by convention.